File: app/src/main/java/com/example/jaguar/weather/model/WeatherObj.java
package com.example.jaguar.weather.model;


public class WeatherObj {
    private String image;
    private String text;

    public WeatherObj(String image, String text) {
        this.image = image;
        this.text = text;
    }

    public String getImage() {
            return image;
        }
        public void setImage(String image) {
            this.image = image;
        }
        public String getText() {
            return text;
        }
        public void setText(String text) {
            this.text = text;
        }

    }

File: app/src/main/java/com/example/jaguar/weather/common/WeatherObj.kt
package com.example.jaguar.weather.common

class WeatherObj(var text: String, var imgUrl: String, var weatherCondition: String, var date: String) {

    var text1: String = ""
        get() = this.text
        set(value) {
            field = value
        }
    var weatherCondition1: String = ""
        get() = this.weatherCondition
        set(value) {
            field = value
        }
    var imgUrl1: String = ""
        get() = "https:"+this.imgUrl
        set(value) {
            field = value
        }
    var date1: String = ""
        get() = this.date
        set(value) {
            field = value
        }
}